### DISCUSSION TITLE
About Gradcam location and demo

### DISCUSSION THREAD

**Discussion by aendrs** [open-mmlab/mmaction2#2353] [Q&A] [answered]
I see that the Gradcam demo disappeared from the new version. Is there a new implementation or suggested way to use it? are there plans for an updated demo?
Thanks

**Comment by cir7** [comment, marked as answer]
we move it to the tools, you can find [here](https://github.com/open-mmlab/mmaction2/blob/main/tools/visualizations/vis_cam.py).
